badgecheck/server/app.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, redirect, render_template, request
+import json
+
+from badgecheck.verifier import verify
+
+from utils import validate_input
+
+app = Flask(__name__)
+
+
+
+
+@app.route("/")
+def home():
+    return render_template('index.html')
+
+
+@app.route("/results", methods=['POST'])
```

Comment:
We do need to use the POST method here, but perhaps it would be better to post to /verify. If you think we should bring it up for improvement, let's treat that as a new issue where we can have targeted discussion.